pom.xml

```diff
@@ -53,6 +53,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <skipTests>false</skipTests>
         <argLine>-Dfile.encoding=UTF-8</argLine>
+
+        <sonar.host.url>http://sonar:9000</sonar.host.url>
```

Comment:
Should this be hard-coded? What if an environment has a different setting? Another option would be to add param when calling mvn:

-Dsonar.host.url=http://sonar:9000